src/new-components/base-field/base-field.tsx

```diff
@@ -94,8 +94,8 @@ type BaseFieldProps = WithEnhancedClassName &
          *
          * Avoid providing interactive elements in the label. Prefer `auxiliaryLabel` for that.
          *
-         * @see secondaryLabel
-         * @see auxiliaryLabel
+         * @see BaseFieldProps['secondaryLabel']
```

Comment:
Maybe we can mention why we want this field to be required, but still allow `null` to be passed in in the description above.

Reply:
Well, technically, we do not do this for every prop in our components that accepts a `React.ReactNode`. And this type allows values such as `null` and `""`.

But yeah, I guess it wouldn't hurt to add a note in the jsdoc. I'll do it.

Reply:
I'm assuming it is because we want to remind ourselves to always have a label associated with the field, but allow it to be omitted if we provide one externally 👀